support/generic_eps_checkout.c

```diff
@@ -16,9 +16,8 @@
 /*
```

Comment:
Currently this code is untestable with real hardware because there is none.
This code is untestable with the simulator because of issues building with the simulator hwlib (but it is not required for testing the sim... the sim can be tested with fsw, the sim terminal, or by other means... so it is not high priority to fix this).
So currently this code seems to have no purpose... but it still seems valuable if one of the two previous statements is resolved, so we want to keep this code as a model for the future.
Agreed upon result:  1. Add some comments in the code to describe this situation.  2.  Write an issue that it is broken with respect to building with the simulator hwlib.

Reply:
Wrote issue https://github.com/nasa/nos3/issues/146.  Once code comment is added, this comment can be resolved.

Reply:
Is there a reason to wait for other issues to be completed if they're reported already?